src/Physics/Resonance/XSection/BostedChristyEMPXSec.cxx

```diff
@@ -0,0 +1,402 @@
+//_________________________________________________________________________
+/*
+ Copyright (c) 2003-2021, The GENIE Collaboration
+ For the full text of the license visit http://copyright.genie-mc.org
+ or see $GENIE/LICENSE
+
+ For the class documentation see the corresponding header file.
+*/
+//_________________________________________________________________________
+#include <TMath.h>
+
+#include "Framework/Algorithm/AlgFactory.h"
+#include "Framework/Algorithm/AlgConfigPool.h"
+#include "Framework/Conventions/GBuild.h"
+#include "Framework/Conventions/Constants.h"
+#include "Framework/Utils/KineUtils.h"
+#include "Framework/ParticleData/PDGLibrary.h"
+#include "Framework/ParticleData/PDGCodes.h"
+#include "Framework/ParticleData/PDGUtils.h"
+#include "Physics/XSectionIntegration/XSecIntegratorI.h"
+#include "Physics/Resonance/XSection/BostedChristyEMPXSec.h"
+
+using namespace genie;
+using namespace genie::constants;
+using namespace genie::utils;
+
+//____________________________________________________________________________
+BostedChristyEMPXSec::BostedChristyEMPXSec() :
+XSecAlgorithmI("genie::BostedChristyEMPXSec")
+{
+  
+}
+//____________________________________________________________________________
+BostedChristyEMPXSec::BostedChristyEMPXSec(string config) :
+XSecAlgorithmI("genie::BostedChristyEMPXSec", config)
+{
+  
+}
+//____________________________________________________________________________
+BostedChristyEMPXSec::~BostedChristyEMPXSec()
+{
+  
+}
+//____________________________________________________________________________
+// resonance cross section fit of electron-proton scattering data
+// sf=0 \sigma_T
+// sf=1 \sigma_L
+double BostedChristyEMPXSec::sigmaR(int sf, double Q2, double W) const
+{
+  double W2 = W*W;
+  // proton mass
+  double Mp = 0.9382727;
+  // pion mass
+  double Mpi = 0.135;
```

Comment:
Also from PDG Library

Reply:
I deliberately took the original values from the code, since a fit was carried out with them.